### ISSUE TITLE
Ignore rule not being applied

### ISSUE THREAD

**Issue by Berkmann18** [remarkjs/remark-lint#190]
I try to get some files ignored (as they are auto-generated) and it still lints them.
Here's what I tried:
```bash
remark -i CHANGELOG.md .
remark . -i CHANGELOG.md
```
```json
"remarkConfig": {
  "settings": {
    "ignore": "CHANGELOG.md"
  }
}
```
```json
"remarkConfig": {
  "settings": {
    "ignore": ["CHANGELOG.md"]
  }
}
```
None of the above work as it still lints `CHANGELOG.md`.

**Comment by Berkmann18** [comment]
@wooorm The thing is is that the `-i` **doesn't** work, it's not being taken into account (as mentioned above).

**Comment by wooorm** [comment]
@Berkmann18 Please click through on the `.remarkignore` files, and how they work.

**Comment by Berkmann18** [comment]
This issue still occurs having `remark -i CHANGELOG.md .` (as well as `remark . -i CHANGELOG.md`) even when the `.remarkignore` is:
```
CHANGELOG.md
```
But it doesn't happen when `-i CHANGELOG.md` is omitted.

**Comment by wooorm** [comment]
The `-i` flag gets a *path* to an ignore file, not an actual file to be ignored. You can use it for example like so: `remark -i .gitignore` to pull patterns from git.
